.eslintrc.js

```diff
@@ -36,18 +36,17 @@ module.exports = {
     'id-length': [
```

Comment:
It'll be good to eventually adapt `@serverless/eslint-config`, e.g. just fact that we're not using _strict mode_ is dangerous, and can result with an errors

But not task for now I believe.